IntegrationTests/UITests/UITests.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="FluentAssertions" Version="5.2.0" />
     <PackageReference Include="NSubstitute" Version="3.1.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />
+    <Reference Include="System.DirectoryServices" />
```

Comment:
What is this needed for?

Reply:
Yes, weird. Because of 83f007423dd7b5e48ff270fb98c12517a155bf26.
Maybe since calling new functions:
- `AsyncTestHelper.WaitForPendingOperations`
- `formBrowse.RevisionGridControl.GetSelectedRevisions().Count`
- something else?!?

Reply:
I can see no imports for this assembly, so it looks sus.
Does it work if you remove this reference?

Reply:
No, it does not: refer to https://ci.appveyor.com/project/gitextensions/gitextensions/builds/30049049.

Reply:
Ahhh, this looks like the culprit
https://github.com/gitextensions/gitextensions/blob/7273784935f964f74be8fe729e782c0122385892/GitUI/UserControls/RevisionGrid/RevisionGridControl.cs#L636

I think we should create our own enum and ditch the assembly reference. 
```cs
    public enum SortDirection
    {
        /// <summary>
        ///  Sort from smallest to largest. For example, A to Z.
        /// </summary>
        Ascending,

        /// <summary>
        //   Sort from largest to smallest. For example, Z to A.
        /// </summary>
        Descending
    }
```

We will reap multiple benefits with such change (better perf, less memory, and smaller deployment footprint in .NET Core).